### PR TITLE
Fix about dialog

### LINKED ISSUE #849: About page does not show LICENSE_LINK
## Bug Description

Screenshot_2019-07-10 Threema Web

## Your Environment

* Threema Web version: Version 2.2.0-beta.3 Glärnisch
* Threema App version: 4.0-beta2 (Android)

### PR DESCRIPTION
**Pull request by dbrgn**
Fixes #849.

### CODE REVIEW

File: src/partials/messenger.ts [hunk 1]
```diff
@@ -1031,9 +1041,12 @@ class NavigationController {
     /**
      * Show dialog.
      */
-    public showDialog(name, ev) {
+    public showDialog(name, ev, controller?) {
```
Comment (lgrahl) [inline src/partials/messenger.ts]: You should be able to use default arguments here. Also, maybe add types.

```
public showDialog(name: string, ev: <no-idea>, controller: DialogController = DialogController) {
```

Comment (dbrgn) [reply src/partials/messenger.ts]: The class `DialogController` is not of type `DialogController` :wink:

Comment (lgrahl) [reply src/partials/messenger.ts]: You're missing the point. :wink:

Comment (dbrgn) [reply src/partials/messenger.ts]: Was about to give up on the `controller` type until I found the `Type` hack.

**Pull request merged**